pkg/providers/aws/cluster_vpc.go

```diff
@@ -457,37 +457,53 @@ func getSubnets(ec2Svc ec2iface.EC2API) ([]*ec2.Subnet, error) {
 	return subs, nil
 }
 
+func getVPCIDByClusterSubnets(ec2Svc ec2iface.EC2API, clusterID string) (*string, error) {
+	var subs []*ec2.Subnet
+	listOutput, err := ec2Svc.DescribeSubnets(&ec2.DescribeSubnetsInput{})
+	if err != nil {
+		return aws.String(""), err
+	}
+	subs = listOutput.Subnets
+	for _, sub := range subs {
+		for _, tag := range sub.Tags {
+			if (*tag.Value == "owned" || *tag.Value == "shared") &&
+				*tag.Key == fmt.Sprintf("%s%s", clusterOwnedTagKeyPrefix, clusterID) {
```

Comment:
`fmt.Sprintf("%s%s", clusterOwnedTagKeyPrefix, clusterID)` is used in multiple places, might be worth creating a little helper func here, just in the future if this tag is to ever change.

Reply:
addressed. Created a helper  function